deployment/merge_aerie_db/merge_db/migrate_sequencing.sql

```diff
@@ -9,16 +9,64 @@ alter table metadata.expansion_rule_tags set schema tags;
 -- Metadata Schema is empty now
 drop schema metadata;
 
--- Update Foreign Keys
+-- Update Triggers
+drop trigger set_timestamp on sequencing.expansion_rule;
+drop function sequencing.expansion_rule_set_updated_at();
+
+create trigger set_timestamp
+before update on sequencing.expansion_rule
+for each row
+execute function util_functions.set_updated_at();
+
+drop trigger set_timestamp on sequencing.user_sequence;
+drop function sequencing.user_sequence_set_updated_at();
+
+create trigger set_timestamp
+before update on sequencing.user_sequence
+for each row
+execute function util_functions.set_updated_at();
+
+-- Update Foreign Keys, handling orphans first
+delete from tags.expansion_rule_tags
+  where not exists(
+   select from tags.tags t
+   where tag_id = t.id);
 alter table tags.expansion_rule_tags
   add foreign key (tag_id) references tags.tags
   on update cascade
   on delete cascade;
+
+delete from sequencing.expanded_sequences
+  where not exists(
+   select from merlin.simulation_dataset sd
+   where simulation_dataset_id = sd.id);
 alter table sequencing.expanded_sequences
   add constraint expanded_sequences_to_sim_run
     foreign key (simulation_dataset_id)
       references merlin.simulation_dataset
       on delete cascade;
+
+update sequencing.expansion_rule
+  set authoring_mission_model_id = null
+  where not exists(
+   select from merlin.mission_model m
+   where authoring_mission_model_id = m.id);
+update sequencing.expansion_rule
+  set authoring_mission_model_id = null,
+      activity_type = null
+  where not exists(
+   select from merlin.activity_type at
+   where (authoring_mission_model_id, activity_type) = (at.model_id, at.name));
```

Comment:
you know what to do

Reply:
also FK

Reply:
done. also updated the comment so its more intuitive why there is no foreign key